packages/storybook/src/schematics/migrate-defaults-5-to-6/migrate-defaults-5-to-6.ts

```diff
@@ -0,0 +1,286 @@
+          name: project,
+          uiFramework:
+            workspaceJson.projects[project].architect.storybook.options
+              .uiFramework,
+          configFolder:
+            workspaceJson.projects[project].architect.storybook.options.config
+              .configFolder,
+        };
+      }
+    });
+
+    return chain(
+      projectsThatHaveStorybookConfiguration
+        .filter(
+          (projectWithStorybook) =>
+            projectWithStorybook && projectWithStorybook.configFolder
+        )
+        .map((projectWithStorybook) => {
+          return chain([
+            migrateStorybookInstance(
+              keepOld,
+              projectWithStorybook.name,
+              projectWithStorybook.uiFramework,
+              projectWithStorybook.configFolder
+            ),
+          ]);
+        })
+    );
+  };
+}
+
+export function migrateStorybookInstance(
+  keepOld: boolean,
+  projectName: string,
+  uiFramework: string,
+  configFolder?: string
+): Rule {
+  return chain([
+    migrateProjectLevelStorybookInstance(
+      keepOld,
+      projectName,
+      uiFramework,
+      configFolder
+    ),
+  ]);
+}
+
+function maybeUpdateVersion(): Rule {
+  return updateJsonInTree('package.json', (json) => {
+    json.dependencies = json.dependencies || {};
+    json.devDependencies = json.devDependencies || {};
+
+    const allStorybookPackagesInDependencies = Object.keys(
+      json.dependencies
+    ).filter((packageName: string) => packageName.startsWith('@storybook/'));
+
+    const allStorybookPackagesInDevDependencies = Object.keys(
+      json.devDependencies
+    ).filter((packageName: string) => packageName.startsWith('@storybook/'));
+
+    const storybookPackages = [
+      ...allStorybookPackagesInDependencies,
+      ...allStorybookPackagesInDevDependencies,
+    ];
+
+    storybookPackages.forEach((storybookPackageName) => {
+      if (json.dependencies[storybookPackageName]) {
+        const version = checkAndCleanWithSemver(
+          storybookPackageName,
+          json.dependencies[storybookPackageName]
+        );
+        if (lte(version, '6.0.0')) {
+          json.dependencies[storybookPackageName] = storybookVersion;
+          needsInstall = true;
+        }
+      }
+      if (json.devDependencies[storybookPackageName]) {
+        const version = checkAndCleanWithSemver(
+          storybookPackageName,
+          json.devDependencies[storybookPackageName]
+        );
+        if (lte(version, '6.0.0')) {
+          json.devDependencies[storybookPackageName] = storybookVersion;
+          needsInstall = true;
+        }
+      }
+    });
+    return json;
+  });
+}
+
+function upgradeStorybookPackagesInPackageJson(): Rule {
+  /**
+   * Should upgrade all @storybook/* packages in package.json
+   */
+  return (tree: Tree, context: SchematicContext) => {
+    return chain([
+      maybeUpdateVersion(),
+      formatFiles(),
+      addInstallTask({ skipInstall: !needsInstall }),
```

Comment:
Looks like this PR doesn't contain my changes I added yesterday.

This line should be changed to

```
() => addInstallTask({ skipInstall: !needsInstall })
```